src/batch/src/rpc/service/task_service.rs

```diff
@@ -119,12 +121,14 @@ impl TaskService for BatchServiceImpl {
             plan,
             epoch,
             tracing_context,
+            captured_context,
         } = req.into_inner();
 
         let task_id = task_id.expect("no task id found");
         let plan = plan.expect("no plan found").clone();
         let epoch = epoch.expect("no epoch found");
         let tracing_context = TracingContext::from_protobuf(&tracing_context);
+        let captured_context = captured_context.expect("no captured context found");
```

Comment:
`time_zone` is optional. Here it seems to be optional because Protobuf always compile a user-defined message to an `Option<>`. But actually, the field is always filled except for testing.